pkgs/by-name/sn/snander/package.nix

```
@@ -0,0 +1,48 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  nix-update-script,
+  libusb1,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "snander";
+  version = "1.7.8";
+
+  src = fetchFromGitHub {
+    owner = "Droid-Max";
+    repo = "SNANDer";
+    tag = "v.${finalAttrs.version}";
+    hash = "sha256-YsNWjlbUTAURD6eOt8ZVDBGMZT4w0hGFO6N42CdX2XA=";
+  };
+
+  buildInputs = [ libusb1 ];
+
+  makeFlags = [
+    "-C"
+    "src"
+  ];
+
+  installFlags = [
+    "DESTDIR=${placeholder "out"}"
+    "PREFIX="
+  ];
+
```

ensures binary was installed correctly, runs and is the right version
```suggestion

  doInstallCheck = true;
  nativeInstallCheckInputs = [ versionCheckHook ];
  versionCheckProgramArg = "-h";

```